[COLAB-2530] Add Back in Missing/Outdated Instructions in ServiceNow (#32507)
DataDog/documentation: content/en/integrations/guide/servicenow-itom-itsm-setup.md

- The field mappings are at the bottom of the record. Some basic mappings are included. This is where you select the fields to include, define the format, and select the target fields in your ServiceNow instance.
{{% /collapse-content %}}

{{% collapse-content title="Transform correlated alert data" level="h4" expanded=false id="transform-correlated-alert-data" %}}
To use information from correlated alerts to populate values in ServiceNow, add a new onBefore transform script under the Datadog Cases ITSM/ITOM table transform map. 

> You must modify your script with the desired fields from data in the correlated alert column so it can populate values in the ServiceNow Incident. Below is a sample script that you customize for your needs:

Reviewer: ```suggestion
To populate data from your alert column into the ServiceNow Incident, you must add the names of the correlated fields to your script. Below is a sample script that you can customize for your needs:
```
Wondering if this makes sense? 🙂 I was concerned that `desired fields from data in the correlated alert column` was a little hard to parse, but I want to make sure it's accurate.
